requirements.txt

```diff
@@ -1 +1,5 @@
--r tools/splat/requirements.txt
\ No newline at end of file
+-r tools/splat/requirements.txt
+mapfile-parser>=1.2.1,<2.0.0
+colorama
+
+spimdisasm>=1.16.3,<2.0.0
```

Comment:
both splat's requirements.txt and this one list spimdisasm, but a slightly different version range. iirc this can confuse pip. Could you check if it works fine?

Reply:
Just did and it works for me (note this is also the same as in AF)